Final Project

ChaoWuu opened the pull request.

Commit:
For homework#1

Commit:
Test of homework#1

Commit:
Test of homework#1

Commit:
update file path

Commit:
Merge branch 'ChaoWuu' of https://github.com/ChaoWuu/applied_ds into ChaoWuu

Commit:
upload example file

ChaoWuu committed:
Add files via upload

ChaoWuu committed:
Merge branch 'master' into ChaoWuu

ChaoWuu committed:
Add files via upload

Commit:
projec#2

Commit:
merge

Commit:
merge upstream

Commit:
final project

Commit:
Merge branch 'master' of https://github.com/em212/applied_ds into ChaoWuu

jzf2101 reviewed (changes requested):
This PR removes your notebook?

ChaoWuu commented:
> This PR removes your notebook?
I replaced my old version with the new version of my notebook, and deleted unnecessary files

ChaoWuu commented:
> > This PR removes your notebook?
> > I replaced my old version with the new version of my notebook, and deleted unnecessary files

I found there are some errors in my nb, I'll revise and recommit

Commit:
revision

Commit:
revision_2
